supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -103,4 +103,28 @@ The following list includes homographs that might commonly appear in technical d
 * object
 * project
 
+[[minimalism]]
+== Minimalism
+Minimalism is a methodology for creating targeted documentation focused on your readers' needs. If you understand your customers' needs, you can write shorter and simpler documentation specific to what customers want to do.
+
+Minimalism has five principles:
+
+=== Customer focus and action orientation
```

Comment:
Just wondering if they should be presented as numbered 1-5, since we're mentioning that there are 5 principles?

Reply:
That's a good idea, @bergerhoffer. If @mportman12 doesn't mind, I'd say make the change.

Reply:
Added numbering. I didn't want to lose the subheading structure, so I kept the formatting and added "Principle #:" to the start of each heading.